http.go

```diff
@@ -72,7 +74,12 @@ type HTTPError struct {
 var websocketHub *WSHub
 
 func ListenAndServe(addr string) error {
+	var err error
 	log.Printf("Starting HTTP service (http://%s)...", addr)
+	staticContent, err = fs.Sub(embeddedFS, "web")
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Since this loads embedded content, maybe we could do it `func init()` (above func ListenAndServe)? .. possibly also panic'ing on any error.

```go
func init() {
  var err error
  staticContent, err = fs.Sub(embeddedFS, "web")
  if err != nil {
    panic(err)
  }
}

func ListenAndServe(addr string) error {
...
```

(Otherwise I would prefer that we return the error from fs.Sub instead of calling log.Fatal, since ListenAndServe already returns an error.)

Reply:
Done